### PR TITLE
ISPN-11072 Move initialization of Elytron security providers to a method

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-11072

### CODE REVIEW

**Comment by wburns** [comment]
It appears there are lot of broken tests now. https://ci.infinispan.org/job/Infinispan/job/PR-7673/1/#showFailuresLink

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/security/ElytronSaslProviders.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.1
+ **/
+public class ElytronSaslProviders {
+   /**
+    * Install the Elytron SASL security providers. Should be invoked before constructing an authentication-enabled
+    * RemoteCacheManager
+    */
+   public static void installSecurityProviders() {
```
Comment (pruivo) [inline client/hotrod-client/src/main/java/org/infinispan/client/hotrod/security/ElytronSaslProviders.java]: I don't understand where this method should be invoked. 
Is the user responsible to invoke it before creating the `RemoteCacheManager`?

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/security/SecurityActions.java [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
    static void addSecurityProvider(Provider provider) {
       doPrivileged(() -> {
+
```
Comment (pruivo) [inline client/hotrod-client/src/main/java/org/infinispan/client/hotrod/security/SecurityActions.java]: nitpicking :) new line can be removed.

**Comment by tristantarrant** [comment]
Closing for inactivity. Please reopen once ready.

**Pull request closed without merging**